python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -599,6 +599,19 @@ def __volumeSummary( plug ) :
 
 		],
 
+		"attributes.volumeStepScale" : [
+
+			"description",
+			"""
+			Raymarching step size is calculated using this scale multipled by the volume voxel size.
+			Note if volumeStepSize overrides this value. 
```

Comment:
Is this really automagic like that, rather than just being a multiplier on the existing step size? The [Arnold docs](https://support.solidangle.com/display/A5AFMUG/Volume) describe it firstly as "A scaling factor applied to the step size. But they also mention an "automatic  step size" checkbox that we don't seem to have in the Gaffer UI, and I can't see in `kick -info volume`. How do all these things relate in practice?

 Also, what does "Note if volumeStepSize overrides this value" mean?